docs/docs/legal/privacy-policy.mdx

```diff
@@ -6,25 +6,26 @@ import PiidTable from './_piid-table.mdx';
 
 ## Introduction
 
-This privacy policy applies to CAOS Ltd., the websites it operates (including caos.ch, zitadel.ch, zitadel.cloud and zitadel.com) and the services and products it provides (including ZITADEL). This privacy policy describes how we process personal data for the provision of this websites and our products.
+This privacy policy applies to CAOS Ltd., the websites it operates (including zitadel.ch, zitadel.cloud and zitadel.com) and the services and products it provides (including ZITADEL). This privacy policy describes how we process personal data for the provision of this websites and our products.
```

Comment:
Do we need to mention ZITADEL and ZITADEL Cloud?

Reply:
I would leave it like that for clarity. We reference this policy from both the Cloud ToS and the Support Program.

Reply:
works with me, just wanted to check if it is needed 👍